lib/plugins/digging.js

```diff
@@ -162,13 +184,13 @@ function inject (bot) {
     function onBlockUpdate (oldBlock, newBlock) {
       // vanilla server never actually interrupt digging, but some server send block update when you start digging
       // so ignore block update if not air
-      // All block update listeners receive (null, null) when the world is unloaded. So newBlock can be null.
```

Comment:
Why did you remove this comment?

Reply:
Can you add the comment back?

Reply:
I have a more updated version of digging.js on the discord. Search for attachments from 'Vakore', unable to do stuff with npm rn.